source/deployment/customize-mattermost.rst

```diff
@@ -0,0 +1,59 @@
+Customizing Mattermost
+======================
+
+There are several ways to customize your Mattermost server. 
+
+If customizing Mattermost, please avoid branding that could be confused with the Mattermost brand. For example, it's okay to brand as "Healthcare Central" because it's a completely different brand. "Mattermost Healthcare Central" is not okay, because it can potentially be confused with the Mattermost brand. Please see the `Mattermost trademark guidelines <https://www.mattermost.org/trademark-standards-of-use/>`_ for details.
+
+While you're welcome to add your own copyright notice in the user interface if you feel it is warranted by your changes, we ask that you do not remove the Mattermost, Inc. copyright notice from the login footer or from the About dialog.
+
+Mattermost Webapp
+-----------------
+
+The Mattermost webapp is licensed under the Apache 2.0 license. To modify and use with the Mattermost server, you can:
+
+1. Install the Mattermost server by following one of our installation guides
+2. Fork the `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`_ repository
+3. Make your changes 
+4. Run ``make package``. This will create ``mattermost-webapp.tar.gz``. 
+5. Copy ``mattermost-webapp-tar.gz`` to the location Mattermost was installed in Step 1. 
+6. Remove the existing ``client`` folder. 
+7. Run ``tar -xvf mattermost-webapp.tar.gz`` to extract your new customized ``client`` folder
+8. Restart your Mattermost server 
+
+In Mattermost Enterprise Edition, it is possible to customize certain parts of the webapp without forking using our :doc:`Custom Branding <../administration/branding>` settings. 
+
+Mattermost Server
+-----------------
+
+There are a few things that can be customized in the Mattermost server without forking. 
+
+1. Modify text in the Mattermost interface by modifying the ``en.json`` file 
```

Comment:
these were taking from IT's old GitHub issue, we may want to check if any of these items need to be updated